docs/contributor/03-10-hyperscaler-account-pool.md

```diff
@@ -82,3 +84,34 @@ metadata:
     tenant-name: {TENANT_NAME}
     hyperscaler-type: "gcp_cf-sa30"
 ```
+
+## Secret Bindings Selection Rules
+
+<!-- rules Overview -->
+HAP evaluates a set of rules to determine what labels to use when querying secret bindings. Input to the rules consists of an SKR's plan, hyperscaler and region. If evaluated to true the rules modify or add labels used in the secret bindings resource query. There are four possible rules to configure:
+* `hap.platformRegionRule` - if evaluated to true the `_<PLATFOR_REGION>` is appended to the `hyperscaler-type` label when searching, refered to as platform region based search,
+* `hap.clusterRegionRule` - if evaluated to true the `_<HYPERSCALER_REGION>` is appended to the `hyperscaler-type` label when searching, refered to as cluster region based search,
+* `hap.sharedRule` - if evaluated to true the `shared: true` label is used when searching, refered to as shared based search,
+* `hap.euAccessRule` - if evaluated to true the `shared: true` label is used when searching, refered to as euAccess based search.
```

Comment:
Are sharedRule and euAccessRule mutually exclusive?

Reply:
We agreed that all the rules can be inclusive because they are independent of each other.

Reply:
So we need to put it here explicitly and possibly add an example.
This means that `shared` and `euAccess` can coexist.

Reply:
```
platformRegionRule='trial:cf-eu11'
clusterRegionRule='trial:eu-central-1'
sharedRule='trial:eu-central-1,eu-central-2'
euAccessRule='cf-eu11'
```

Reply:
No, the rules is not valid because `sharedRule='trial:eu-central-1,eu-central-2' contains two regions for one plan, coma is not used for separating entries and eu-central-2 is left alone.

Reply:
So how we can specify list of respective regions? `trial:eu-central-1, trial:eu-central-2`?

Reply:
Discussed, ok.